wp_localization/src/lib.rs

```diff
@@ -0,0 +1,199 @@
+use fluent_bundle::FluentValue;
+use fluent_templates::Loader;
+use std::{collections::HashMap, fmt::Debug, fmt::Display};
+use strum_macros::{EnumIter, IntoStaticStr};
+
+fluent_templates::static_loader! {
```

Comment:
I was hoping we could implement this ourselves so that we can get rid of this third-party library. I guess it could be one of the next steps.

Reply:
I haven't looked into this at all. So, although I agree with you that it'd be good not to rely on a 3rd party, I don't think it's very impactful to take this on at the moment.